web3sTests/Account/EthereumAccountTests.swift

```diff
@@ -28,6 +28,12 @@ class EthereumAccountTests: XCTestCase {
         let account = try? EthereumAccount.create(keyStorage: EthereumKeyLocalStorage(), keystorePassword: "PASSWORD")
         XCTAssertNotNil(account, "Failed to create account. Ensure key is valid in TestConfig.swift")
     }
+
+	func testImportAccount() {
+		let account = try? EthereumAccount.importAccount(keyStorage: EthereumKeyLocalStorage(), privateKey: "0x2639f727ded571d584643895d43d02a7a190f8249748a2c32200cfc12dde7173", keystorePassword: "PASSWORD")
+
+		XCTAssertNotNil(account, "Failed to import account.")
```

Comment:
Sorry for nitpick, but better to check account?.address.value == "0x..." instead of nil to be sure it was imported correctly

Reply:
Ok perfect ;)